modules/cluster/src/main/java/io/fluo/cluster/runner/ScanOptions.java

```diff
@@ -40,6 +40,10 @@
   @Parameter(names = {"-h", "-help", "--help"}, help = true, description = "Prints help")
   public boolean help;
 
+  @Parameter(names = {"-esc", "--escape-non-ascii"}, help = true,
+      description = "Hex encode non ascii bytes")
+  public boolean hexEncNonAscii = false;
```

Comment:
I made a changes in e1ff9a1 to use the arity option and set the default to true.  I tested it with webindex and it works as expected.